src/stack_length.js

```diff
@@ -21,18 +21,19 @@ this.items = [];
 // push(element){
 // this.items.push(element);
 // }
-clear()  {
-  this.items = [];
-}
-size(){
-return this.items.length;
-if(this.items != []){
-
-return true;
-}
-else{
-return false;
-}
+// clear()  {
+//   this.items = [];
+// }
+length(){
+return Stack.items.length;
+console.log(length())
```

Comment:
Generally I've been getting feedback about removing console logs before submission. 

Also, anything you put after a return statement will not run; `return` will end a function